build/Dockerfile.operator

```diff
@@ -9,4 +9,6 @@ COPY deploy/handler/service_account.yaml /bindata/kubernetes-nmstate/rbac/
 COPY deploy/handler/role.yaml /bindata/kubernetes-nmstate/rbac/
 COPY deploy/handler/role_binding.yaml /bindata/kubernetes-nmstate/rbac/
 
+ENV ENABLE_OVS=""
```

Comment:
Is this correct? AFAIK we currently don't have OVS installed in knmstate CIWhy is this here? AFAIK we currently don't have OVS installed in knmstate CI

Reply:
This is the Dockerfile for openshift so it does not affect the CI

Reply:
Fixed I was all wrong